SonarQube.TeamBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -50,16 +51,21 @@ public ProcessedArgs(string key, string name, string version, bool installLoader
             {
                 throw new ArgumentNullException("globalFileProperties");
             }
+            if (scannerEnvProperties == null)
+            {
+                throw new ArgumentNullException("scannerEnvProperties");
```

Comment:
It is better to use `nameof(variable)` instead of plain string to avoid any typo.